src/handler/authentication.rs

```diff
@@ -36,3 +39,44 @@ https://link/{jwt}"
 
     Ok(StatusCode::CREATED)
 }
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SignUp {
+    pub user_name: String,
+    pub password: String,
+    pub token: String,
+}
+
+impl Validator for SignUp {
+    fn validate(&self) -> anyhow::Result<()> {
+        let rules = vec![
+            ("user_name", RuleType::UserName),
```

Comment:
ここ"user_name" であってるの？
```suggestion
            (&self.user_name, RuleType::UserName),
```